ConfigSpace/hyperparameters.pyx

```diff
@@ -1138,7 +1632,8 @@ cdef class UniformIntegerHyperparameter(IntegerHyperparameter):
 cdef class NormalIntegerHyperparameter(IntegerHyperparameter):
     cdef public mu
     cdef public sigma
-    cdef nfhp
+    cdef public nfhp
+    cdef normalization_constant
```

Comment:
Is there a reason we don't have this for the normal float hyperparameter?

Reply:
The normalization constant? The continous distribution is already normalized per default, whereas the discrete one is not. 

If we wanted the build-in normalization in the discrete case, we'd have to use a binomial. However, since a continuous normal was used before to power the normal integer HP, I decided to stick with what was already there.